apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -46,6 +47,87 @@ import {
 import RequiresConfirmationController from "./RequiresConfirmationController";
 import { DisableAllEmailsSetting } from "./settings/DisableAllEmailsSetting";
 
+const SingleUseLinksManager = ({ team }: Pick<EventTypeSetupProps, "team">) => {
+  const formMethods = useFormContext<FormValues>();
+  const { t } = useLocale();
+  const [animateRef] = useAutoAnimate<HTMLUListElement>();
+  return (
+    <Controller
+      name="singleUseLinks"
+      control={formMethods.control}
+      render={({ field: { value, onChange } }) => {
+        if (!value) {
+          value = [];
+        }
+        const addSingleUseLink = () => {
+          const newSingleUseLink = generateHashedLink(formMethods.getValues("users")[0]?.id ?? team?.id);
+          if (!value) value = [];
+          value.push(newSingleUseLink);
+          onChange(value);
+        };
+
+        const removeSingleUseLink = (index: number) => {
+          if (!value) value = [];
+          value.splice(index, 1);
+          onChange(value);
+        };
+
+        return (
+          <ul ref={animateRef}>
+            {value &&
+              value.map((val: string, key: number) => {
+                const singleUseURL = `${WEBSITE_URL}/d/${val}/${formMethods.getValues("slug")}`;
```

Comment:
These links does not work for org domains, it throws 404 page does not exist. I know the implementation is same as that of private links, but I found that it is bug there too. 

https://github.com/calcom/cal.com/blob/aab9193e7bce6f87156340f5c8c151987ffdc54a/apps/web/components/eventtype/EventTypeSingleLayout.tsx#L313-L315

You should generate the URL the same way we are doing it for Copy link to event, at the top besides the save button

Reply:
@Amit91848 How can we get the org URLs to run on local? 😅 

For me any org URLs are not working on local.
But yeah anyhow I have fixed the domain part which was prominently visible, but not sure any other fix is needed or not. And also couldn't test that's why..

Reply:
You can follow the steps in the readme here https://github.com/calcom/cal.com/tree/main/packages/features/ee/organizations.

Reply:
Thanks Amit!